### PR TITLE
apidocs

### PR DESCRIPTION
**Pull request by mosquito**

### CODE REVIEW

File: aiomisc/entrypoint.py [hunk 1]
```diff
@@ -136,24 +142,31 @@ async def __aenter__(self) -> "Entrypoint":
...
+    ) -> None:
         try:
             if self.loop.is_closed():
                 return
 
             await self.graceful_shutdown(exc_val)
             self.shutting_down = True
         finally:
-            self.ctx.close()
+            if self.ctx is not None:
```
Comment (dizballanze) [inline aiomisc/entrypoint.py:155]: `if self.ctx:`

File: aiomisc/service/tls.py [hunk 2]
```diff
@@ -96,6 +98,8 @@ async def start(self):
             ssl=ssl_context,
         )
 
-    async def stop(self, exc: Exception = None):
+    async def stop(self, exc: Exception = None) -> None:
```
Comment (dizballanze) [inline aiomisc/service/tls.py:101]: Optional[Exception]

File: aiomisc/service/tracer.py [hunk 3]
```diff
@@ -84,13 +85,16 @@ def log_diff(self, diff):
...
+
         differences = self.compare_snapshot(
             self._snapshot_on_start,
             self.take_snapshot(),
         )
 
         return self.log_diff(differences)
 
-    async def stop(self, exception: Exception = None):
+    async def stop(self, exception: Exception = None) -> None:
```
Comment (dizballanze) [inline aiomisc/service/tracer.py:99]: Optional[Exception]

**Review by dizballanze — APPROVED** [review]

File: aiomisc/service/tls.py (hunk 2 above)
Comment (mosquito) [reply aiomisc/service/tls.py:101]: This notation means the same

File: aiomisc/entrypoint.py (hunk 1 above)
Comment (mosquito) [reply aiomisc/entrypoint.py]: done

**Pull request merged**